js/components/MilkdropWindow/index.js

```diff
@@ -36,6 +36,8 @@ class MilkdropWindow extends React.Component {
       "butterchurn"
     );
 
+    this._handleKeyboardInput = this._handleKeyboardInput.bind(this);
```

Comment:
This should probably go in the constructor.

Reply:
oh whoops, missed that in your first comment :-X